newdle/templates/base_email.html

```diff
@@ -0,0 +1,48 @@
+<table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+    style="margin-top: 20px; line-height: 1.7; color: #555;">
+    <tr>
+        <td>
+            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                style="max-width: 660px; font-family: Helvetica, Arial, sans-serif; font-size: 14px; background: #FFFFFF;">
+                <tr>
+                    <td style="border: 1px solid #ddd;">
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                            style="border-collapse: collapse;">
+                            <tr>
+                                <td style="padding: 20px 20px 10px; text-align:left;">
+                                    <img src="https://raw.githubusercontent.com/indico/newdle/master/newdle/client/src/images/logo_color.svg"
```

Comment:
please don't hotlink images from github (or any other external source).

instead, embed it from the actual newdle instance (copy the logo into the `newdle/client/public` folder so it's accessible from outside the react app)

or attach it to the mail itself and use a cid url to access the embedded image

Reply:
Not sure how to embed it from the instance. I copied the image in ```newdle/client/public``` folder but I don't know what to put in the <img ...> field.

Reply:
huh strange, I thought it's super common to have embedded images. Anyway, as long as the mail looks  OK without the external image having been loaded that's fine.

`app.add_url_rule('/logo_color.svg', 'logo', build_only=True)` in app.py and then use `url_for('logo', _external=True)`

Reply:
Solved, thank you so much for the help :smiley: Here there are some refs about the problem, just to know :)
[Reference 1](https://blog.mailtrap.io/embedding-images-in-html-email-have-the-rules-changed/)
[Reference 2](https://sendgrid.com/blog/embedding-images-emails-facts/)

![table](https://user-images.githubusercontent.com/71437950/95168303-63ce4380-07b1-11eb-8c54-a3f862532ea5.png)


Basically for CID attachments, or embedding an image using MIME object there are issues with displaying in web-based email clients. Inline embedding, or base64 images in emails are completely blocked in Outlook.